pages/partner.vue

```diff
@@ -63,6 +63,35 @@
                     </div>
                     <span v-if="extendContractTextVisible">Wysłano prośbę o przedłużenie umowy</span>
                 </div>
+                <div class="card" v-if="contractExtensionOfferVisible">
+                    <div class="section">
+                        <span class="caption">Ważność umowy:</span>
+                        <span class="important">do {{ endDate?.getDate() }}.{{ endDate?.getMonth() + 1 }}.{{ endDate?.getFullYear() }}r</span>
+                    </div>
+                    <div class="section">
+                        <span class="caption">Stawka za godzinę oglądalności:</span>
+                        <span class="important">{{ rate }} zł</span>
+                    </div>
+                    <div class="section">
+                        <span class="caption">Procent otrzymywanych dotacji:</span>
+                        <span class="important">{{ donationPercentage }}%</span>
+                    </div>
+                    <div class="buttons">
+                        <button-component
+                            text="Zaakceptuj ofertę"
+                            icon-name="duplicate-outline"
+                            :on-click="acceptContract"
+                        />
+                        <button-component
+                            text="Odrzuć ofertę"
+                            icon-name="exit-outline"
+                            color="#dd3333"
+                            :on-click="declineContract"
+                        />
```

Comment:
fajnie byłoby dodać jakąś cechę, która odróżni ten komponent od normalnego widoku oferty. np. przyciski innego koloru, jakiś tekst, dodatkowe elementy typu notyfikacja lub kolory pewnych elementów przyciągające wzrok itd - trzeba się wykazać kreatywnością i estetyką

Reply:
zmieniono, myslę że to szczyt moich mozliwości